### PR TITLE
Setup linters

### PR DESCRIPTION
**Pull request by Christelle-12**
## Hello in this PR I :

- Created a new Rails app called 'hellorailsreact'.

- Used the commands  described in the previous lesson's tutorial.

- Ensured that the jsbundling-rails gem is added to your Gemfile.

- Initialized my project with Git.
- Installed needed node dependencies as described in the previous lesson's tutorial.
- Created a babel.config.js in the root folder of my project as described in the previous lesson's tutorial.
- Updated webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
- Ensured my project has a Postgres database set up.
- Created a table for storing  5 different greetings and added a button to help change the message randomly
- Create d an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
- Create a static view that will be the root of your app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,176 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (zainsadaqat) [inline README.md]: Dear @Christelle-12 
You should delete the live demo section if you decide not to deploy the application. If project requirements say you should deploy, they should put the link.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,176 @@
...
+
+<!-- Video recording -->
+
+## 🚀 Live Demo <a name="video-record"></a>
+
+- [Video recording ]()
```
Comment (zainsadaqat) [inline README.md]: ### [OPTIONAL]

Dear @Christelle-12 
You should delete the Video Recording section if you decide not to record the video presentation of the application. If project requirements say you should record, then you should put the link.

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
## Hello @Christelle-12   👋 🙂

You're doing a great job on your project. We're proud of you. Good job so far! 

It's OK to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job. 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Flow
  ✅  Descriptive Pull Request
  ✅  No linter errors

## Required Changes ♻️

- [ ] 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @Christelle-12,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**